CHANGELOG.md

```diff
@@ -1,9 +1,39 @@
 # Unreleased
 
+# 9.13.0 (Feature release)
+
+## New features
+
+### Make Sass errors clearer to users
+
+Previously, the Sass app continued to run even after Sass stopped compiling because of an error.
```

Comment:
This section is great, but there's a couple of things that I think could be clearer:

1. I'm not sure a user would know what we mean by 'Sass app'
2. Does saying 'previously, the Sass app continued to run' suggest that the new behaviour is that it will stop running?

This might be a little long, but what about something like:

```suggestion
Previously, it wasn't obvious to users if Sass has stopped updating because of an error. An error would be printed to the Terminal, but nothing would happen in the browser. We know that it was easy to miss, and could be confusing.
```

Reply:
@36degrees Your suggestion looks good! Just a couple of questions about:

> We know that it was easy to miss, and could be confusing.

- By "it," do we mean the error notification or the error itself?
- What do we mean by "confusing" - the error / error notification or the user experience of Sass not updating without their knowing why?

Could a slight rewording like the below work?

> Previously, it was not obvious to users if Sass has stopped updating because of an error. An error would be printed to the Terminal, but nothing would happen in the browser. We know that this error was easy to miss, and could cause confusion.